test/components/composables/useFormAction.spec.ts

```diff
@@ -24,4 +25,24 @@ describe( 'useFormAction', () => {
 
 		expect( formAction.value ).toBe( expectedAction );
 	} );
+
+	test.each( [
```

Comment:
If you want you can integrate this test and the previous one, the ones from previous would need to be set to use the PAYPAL payment method.

Reply:
Done.